doc/classes/BaseButton.xml

```diff
@@ -49,6 +49,7 @@
 		</member>
 		<member name="button_group" type="ButtonGroup" setter="set_button_group" getter="get_button_group">
 			The [ButtonGroup] associated with the button. Not to be confused with node groups.
+			[b]Note:[/b] The button may be configured as a radio button if a [ButtonGroup] is assigned to it. See [ButtonGroup] for how to configure it.
```

Comment:
```suggestion
			[b]Note:[/b] The button will be configured as a radio button if a [ButtonGroup] is assigned to it. See [ButtonGroup].
```

This was fine before. The button definitely will act as a radio button, that's the whole point here and what the description of `ButtonGroup` tells us.

Reply:
I just feel that it has no effect when `BaseButton.toggle_mode` is not enabled.